src/main/java/org/minions/devfund/noemiguzman/battleship/Ocean.java

```diff
@@ -0,0 +1,201 @@
+     * new ships method.
+     * @return ship list
+     */
+    private Ship[] newShips() {
+        Ship[] ships = new Ship[NRO_SHIPS];
+        for (int i = 0; i < NRO_SHIPS; i++) {
+            ships[i] = selectShip(i);
+        }
+        return ships;
+    }
+
+    /**
+     * get a ship.
+     * @param i int
+     * @return Ship object
+     */
+    private Ship selectShip(int i) {
+
+        final int iLC = 4;
+        final int iDES = 7;
+
+        Ship ship1;
+        switch (i) {
+            case 0:  ship1  = new BattleShip();
+                break;
+            case 1:  ship1  = new BattleCruiser();
+                break;
+            case 2: case iLC - 1: ship1 = new Cruiser();
+                break;
+            case iLC: case iLC + 1:  ship1 = new LightCruiser();
+                break;
+            case iDES - 1: case iDES: case iDES + 1: ship1 = new Destroyer();
+                break;
+
+            default: ship1 = new Submarine();
+                break;
+        }
+        return ship1;
+    }
+    /**
+     * Returns true if the given location contains a ship, false if it does not.
+     * @param row number
+     * @param column number
+     * @return true if it is occupied.
+     */
+    public boolean isOccupied(int row, int column) {
+
+        return !ships[row][column].getShipType().equals("empty");
+    }
+
+    /**
+     * Returns true if the given location contains a real ship, still afloat, (not an EmptySea), false if it does not.
+     * In addition, this method updates the number of shots that have been fired, and the number of hits.
+     * If a location contains a real ship, shootAt should return true every time the user shoots at that same location.
+     * Once a ship has been sunk, additional shots at its location should return false.
+     * @param row number
+     * @param column number
+     * @return true
+     */
+    public boolean shootAt(int row, int column) {
+        this.shotsFired++;
+        if (isOccupied(row, column)) {
+            if (ships[row][column].shootAt(row, column)) {
+                if (ships[row][column].isSunk()) {
+                    shipsSunk++;
+                }
+                hitCount++;
+                return true;
+            }
+            return false;
+        }
+        return false;
+    }
+
+    /**
+     * Prints the ocean. To aid the user, row numbers should be displayed along the left edge of the array.
+     * and column numbers should be displayed along the top.
+     * Numbers should be 00 to 19, not 1 to 20.
+     * The top left corner square should be 0, 0.
+     * Use ’S’ to indicate a location that you have fired upon and hit a (real) ship,
+     * ’-’ to indicate a location that you have fired upon and found nothing there,
+     * ’x’ to indicate a location containing a sunken ship,
+     * and ’.’ (a period) to indicate a location that you have never fired upon.
+     */
+    public void print() {
+        System.out.println(toString());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)